src/lucky_web/tags/select_helpers.cr

```diff
@@ -0,0 +1,24 @@
+module LuckyWeb::SelectHelpers
+  alias SelectOption = (String | Int32 | Int64)
+
+  def select(field : LuckyRecord::AllowedField, **html_options)
```

Comment:
Could you rename this to `select_input`? I think that will result in less conflicts in the future